pims/tests/test_common.py

```diff
@@ -570,13 +571,20 @@ def setUp(self):
         self.v = self.klass(self.filename, **self.kwargs)
         self.expected_shape = shape
         self.expected_len = 5
+        with zipfile.ZipFile('test.zip', 'w') as archive:
```

Comment:
Can you use a tempdir/tempfile https://docs.python.org/3.5/library/tempfile.html for this?